shiny/express/ui/_cm_components.py

```diff
@@ -649,7 +649,7 @@ def card_footer(
 def accordion(
     *,
     id: Optional[str] = None,
-    open: Optional[bool | str | list[str]] = None,
+    open: Optional[bool | str | Iterable[str]] = None,
```

Comment:
Please use `ListOrTuple` type from `../../types` ([link](https://github.com/posit-dev/py-shiny/blob/fcdc2d9efd69279556f405307308c4c7706a4110/shiny/types.py#L50))

It is a little more restrictive than a generic `Iterable`